css/style.css

```diff
@@ -0,0 +1,42 @@
+*{
+    font-size:24px;
```

Comment:
Kindly consider applying responsive units (e.g. em or rem)

Reply:
- [ ] Great job using the inline comments feature for your comments. However, it would be helpful if you could provide an explanation or reason for each requested change. This ensures that the person receiving the feedback understands the reasoning behind the suggested changes and can improve their work accordingly. Thank you for your attention to this important detail! 😊